solr/contrib/ltr/src/java/org/apache/solr/ltr/rest/ManagedFeatureStore.java

```diff
@@ -115,9 +114,6 @@ public synchronized void addFeature(String name, String type,
 
     log.info("register feature {} -> {} in store [" + featureStore + "]",
         name, type);
-    if (!NameValidator.check(name)) {
-      throw new InvalidFeatureNameException(name);
```

Comment:
I thought that originally, but then I looked at that exception and it is also used as a way to let  a user know that the feature name has already been used which i s still a valid case